storyboardOutput/ViewController.swift

```diff
@@ -15,6 +15,5 @@ class ViewController: UIViewController {
         // Do any additional setup after loading the view.
     }
 
-
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/TakumiHasegawa21/storyboardOutput/blob/dd14f2864f08f67bebf563b10bbeced5556d8b8a/storyboardOutput/ViewController.swift#L19"></span>Files should have a single trailing newline<br />
<code>trailing_newline</code> <code>ViewController.swift:19</code></td>
    </tr>
  </tbody>
</table>